packages/config/config/devices/0x019a/11_04_022.json

```diff
@@ -0,0 +1,233 @@
+		{
+			"productType": "0x0004",
+			"productId": "0x000B"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"2": {
+			"label": "LED Alarm Event Reporting",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "LED Off",
+					"value": 0
+				},
+				{
+					"label": "LED On",
+					"value": 1
+				}
+			]
+		},
+		"4": {
+			"label": "Temperature Reporting",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "On",
+					"value": 1
+				}
+			]
+		},
+		"5": {
+			"label": "Temperature Reporting Unit",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Celsius",
+					"value": 0
+				},
+				{
+					"label": "Fahrenheit",
+					"value": 1
+				}
+			]
+		},
+		"6": {
+			"label": "Temperature Alarms",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "On",
+					"value": 1
+				}
+			]
+		},
+		"7": {
+			"label": "High Temperature Alarm Level",
+			"valueSize": 1,
+			"unit": "°C",
+			"minValue": -20,
+			"maxValue": 80,
+			"defaultValue": 40
+		},
+		"8": {
+			"label": "Low Temperature Alarm Level",
+			"valueSize": 1,
+			"unit": "°C",
+			"minValue": -20,
+			"maxValue": 80,
+			"defaultValue": 5
+		},
+		"12": {
+			"label": "Leakage/Moisture Alarm",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "On",
+					"value": 1
+				}
+			]
+		},
+		"13": {
+			"label": "Leakage/Moisture Alarm Level",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 100,
+			"defaultValue": 10
+		},
+		"14": {
+			"label": "Leakage/Moisture Reporting Period",
+			"valueSize": 1,
+			"unit": "hours",
+			"minValue": 0,
+			"maxValue": 120,
+			"defaultValue": 0
+		},
+		"15": {
+			"label": "Activate Supervision",
+			"description": "Activate Supervision command for only important alarm events or all events. *Supervised commands require a confirmation from the gateway when a notification is received.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 1,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "Alarm Report (*)",
+					"value": 1
+				},
+				{
+					"label": "All Reports",
+					"value": 2
+				}
+			]
+		},
+		"20": {
+			"label": "Wake-Up Moisture Pooling Workaround for Fibaro Gateway Users",
```

Comment:
Please help me with a better name here if this does not make sense. Took it from the manufacturer manual